backend/clickhouse/logs.go

```diff
@@ -185,16 +194,40 @@ func (client *Client) LogsKeys(ctx context.Context, projectID int) ([]*modelInpu
 func (client *Client) LogsKeyValues(ctx context.Context, projectID int, keyName string) ([]string, error) {
 	sb := sqlbuilder.NewSelectBuilder()
 
-	// TODO(et) - this is going to be a mess when we add other reserved keys like Trace. Clean this up when that happens.
-	if keyName == "level" {
+	switch keyName {
+	case modelInputs.ReservedLogKeyLevel.String():
 		sb.Select("SeverityText level, count() as cnt").
 			From("logs").
 			Where(sb.Equal("ProjectId", projectID)).
 			Where(sb.NotEqual("level", "")).
 			GroupBy("level").
 			OrderBy("cnt DESC").
 			Limit(50)
-	} else {
+	case modelInputs.ReservedLogKeySecureSessionID.String():
+		sb.Select("SecureSessionId secure_session_id, count() as cnt").
+			From("logs").
+			Where(sb.Equal("ProjectId", projectID)).
+			Where(sb.NotEqual("secure_session_id", "")).
+			GroupBy("secure_session_id").
+			OrderBy("cnt DESC").
+			Limit(50)
```

Comment:
nit: limit value could be `Limit / 2`?

Reply:
Good idea. I made a separate `const` since the existing `Limit` should probably only apply to log lines.
4ba4931279db508261a368b1007dd6b93815e3f8